### PR TITLE
Add default_scope and version bump

### PR DESCRIPTION
**Pull request by spelfrey**
Add default_scope ordering to
-- answers
-- questions
-- survey questions
-- surveys

Version bump

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -29,3 +29,6 @@ testbed
 surveys/fixtures/*.yml
 .rvmrc
 .sass-cache/**/*.sassc
+
+## RubyMine
+.idea/
```
Comment (mellowfish) [inline .gitignore]: Blocking: remove this. (add a `.gitignore` file in the `.idea` folder with `*` in it to hide that folder locally)

Comment (spelfrey) [inline .gitignore]: Interesting. That works.

**Comment by spelfrey** [comment]
The answers are being included in the response set  in the surveyor_controller_method on line 179. I do not see any easy way to alter the sort order of there. Do you have any suggestions?

**Comment by mellowfish** [comment]
:shipit:

**Comment by gray-sanborn-work** [comment]
:shipit:

**Comment by spelfrey** [comment]
default_scope only works on ActiveRecord. These classes include ActiveModel functionality. Thank you @mellowfish for showing me how to run this locally!

**Comment by spelfrey** [comment]
ActiveSupport::Concern provides a way to add default_scope. New commit incoming.

**Comment by spelfrey** [comment]
@mellowfish & @gray-sanborn-work Would you mind looking at this one more time? It works locally. I limited it to answers. When I added it to surveys, survey_sections and questions I got some unexpected results. We'll make those better as needed.

**Comment by mellowfish** [comment]
Still :shipit:

**Comment by gray-sanborn-work** [comment]
**Blocking:** Leaving the gem in an inconsistent state doesn't fly.

**Comment by gray-sanborn-work** [comment]
:shipit:

**Pull request merged**